src/radiation_system.hpp

```diff
@@ -63,6 +63,12 @@ template <typename problem_t> struct RadSystem_Traits {
 	static constexpr amrex::GpuArray<double, Physics_Traits<problem_t>::nGroups + 1> radBoundaries = {0., inf};
 };
 
+// A struct to hold the results of the ComputeRadPressure function.
+struct RadPressureResult {
+	quokka::valarray<double, 4> F;
```

Comment:
Can you add a comment that these are components of the radiation pressure tensor?